entrypoint.sh

```diff
@@ -3,4 +3,4 @@
 zip -r /tmp/training.zip /src
 mkdir -p /tmp/hpf/
 
-gunicorn --pythonpath /src -b 0.0.0.0:"$SERVICE_PORT" --workers=1 -k sync -t "$SERVICE_TIMEOUT" flask_endpoint:app
+gunicorn --pythonpath /src -b 0.0.0.0:6006 --workers=1 -k sync -t 900 flask_endpoint:app
```

Comment:
Our standard practice is to keep them as environment variables, so that they can be changed easily if we want.